java/code/src/com/redhat/rhn/domain/notification/types/UpdateAvailable.java

```diff
@@ -98,7 +98,7 @@ public String getReleaseNotes() {
      */
     public String getReleaseNotesUrl() {
         return isUyuni ? "https://www.uyuni-project.org/pages/stable-version.html" :
-                "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/" +
+                "https://www.suse.com/releasenotes/x86_64/multi-linux-manager/" +
```

Comment:
This is correct! This is what I agreed with @jcayouette . Thank you

Reply:
@jcayouette was working on redefining url paths,  so he is more familiar of the workflow. Joseph can you please confirm if we are good with new url and you will take care that our new release notes will have this path? 
@deneb-alpha fyi